snapcraft.yaml

```diff
@@ -71,6 +91,10 @@ parts:
       - python3-dev
       - libperl-dev
       - libluajit-5.1-dev
+    stage-packages:
```

Comment:
Curious why these need to be added?

Reply:
snapcraft stopped to blindly add stage-packages for all the -dev libs defined in build-packages to allow more fine grained contol about what is shipped....